<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AE20125gui</class>
 <widget class="QMainWindow" name="AE20125gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AE20125gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_con">
      <property name="title">
       <string>Connection</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_con_port">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_con_port">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_con_connect">
         <property name="text">
          <string>&amp;Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_con_status">
         <property name="text">
          <string>Not connected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_con_about">
         <property name="text">
          <string>About</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_freq">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Frequency</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBox_m10">
         <property name="maximum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_m1">
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Label_freq_MHz">
         <property name="text">
          <string>MHz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_k100">
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_k10">
         <property name="maximum">
          <number>9</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_k1">
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_freq_KHz">
         <property name="text">
          <string>KHz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_100">
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_10">
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_1">
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_freq_decimal">
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_01">
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_freq_Hz">
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_wav">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Waveform</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton_wav_sine">
           <property name="text">
            <string>Sine (10MHz)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_wav_tri">
           <property name="text">
            <string>Triangle (2.5MHz)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_wav_rect">
           <property name="text">
            <string>Square (2.5MHz)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_mode">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Mode</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButton_mode_norm">
           <property name="text">
            <string>Normal</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_mode_sweep">
           <property name="text">
            <string>Sweep</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_mode_mod">
           <property name="text">
            <string>Modulation</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_set">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_set_wav">
           <item>
            <property name="text">
             <string>Sine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Square</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_set_frq">
           <property name="text">
            <string>Startup Frequency (Hz)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_set_wav">
           <property name="text">
            <string>Startup waveform</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_set_cal">
           <property name="text">
            <string>Calibration offset</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_set_cal">
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_set_frq">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>10000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_file">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Save/Load</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_file_preset">
           <property name="text">
            <string>Preset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_file_load">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_file_save">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_mod">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Modulation</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_mod_typ">
           <property name="title">
            <string>Type</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_mod_typ_deg">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>359.899999999999977</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_mod_typ_freq">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>10000000.000000000000000</double>
              </property>
              <property name="value">
               <double>10000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_mod_typ_fsk">
              <property name="text">
               <string>FSK</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_mod_typ_phase">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>PSK phase</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_mod_typ_freq">
              <property name="text">
               <string>FSK frequency</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_mod_typ_PSK">
              <property name="text">
               <string>PSK</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_mod_typ_hz">
              <property name="text">
               <string>Hz</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_mod_typ_deg">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>°</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_mod_src">
           <property name="title">
            <string>Source</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="3">
             <widget class="QLabel" name="label_mod_src_hz">
              <property name="text">
               <string>Hz</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_mod_src_int">
              <property name="text">
               <string>Internal</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_mod_src_ext">
              <property name="text">
               <string>External</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_mod_src_freq">
              <property name="text">
               <string>Internal Frequency:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_mod_src_freq">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>100.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_sweep">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Sweep</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label_sqeep_stop">
           <property name="text">
            <string>Stop Frequency</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_sweep_start">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>10000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_sweep_mode">
           <property name="text">
            <string>Sweep Mode</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_sweep_stop">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>10000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_sweep_start">
           <property name="text">
            <string>Start Frequency</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_sweep_freq">
           <property name="text">
            <string>Sweep Frequency</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_sweep_freq">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_sweep_hzstart">
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_sweep_hzstop">
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_sweep_hzsweep">
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="radioButton_sweep_loop">
           <property name="text">
            <string>Loop</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QRadioButton" name="radioButton_sweep_swing">
           <property name="text">
            <string>Swing</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_pll">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>PLL Reference</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="checkBox_pll_on">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_pll_fact">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Factor:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_pll_fact">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999.899999999999977</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_pll_off">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Offset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_pll_off">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-1000000000</number>
           </property>
           <property name="maximum">
            <number>1000000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_pll_hz">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_hw">
      <property name="title">
       <string>Hardware/Firmware Revision</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_hw_pid">
         <property name="text">
          <string>Product ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_hw_pidv">
         <property name="text">
          <string>unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_hw_hw">
         <property name="text">
          <string>Hardware</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_hw_hwv">
         <property name="text">
          <string>unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_hw_fw">
         <property name="text">
          <string>Software</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_hw_fwv">
         <property name="text">
          <string>unknown</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_debug">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;DEBUG:&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
